app/models/search/bookmark_indexer.rb

```diff
@@ -18,25 +21,34 @@ def self.index_all(options={})
   def self.mapping
     {
       "bookmark" => {
-        "_parent" => {
-          type: 'bookmarkable'
-        },
-        properties: {
-          bookmarkable_type: {
-            type: 'string',
-            index: 'not_analyzed'
+        "properties" => {
```

Comment:
I suppose this could use the new Ruby 1.9 syntax like the equivalent block in the Works indexer